### PR TITLE
add IsNotMountPoint() to mount_unsupported.go

### PR DESCRIPTION
**Pull request by rootfs**
fix the cross build issue

``` release-note
cross: add IsNotMountPoint() to mount_unsupported.go
```

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/35566)

### CODE REVIEW

**Comment by ixdy** [comment]
@k8s-bot cross build this

**Comment by ixdy** [comment]
@k8s-bot build this

**Comment by ixdy** [comment]
x-ref #33616

File: pkg/util/mount/mount_unsupported.go [hunk 1]
```diff
@@ -57,3 +57,7 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 func (mounter *SafeFormatAndMount) diskLooksUnformatted(disk string) (bool, error) {
 	return true, nil
 }
+
+func IsNotMountPoint(file string) (bool, error) {
+	return true, nil
```
Comment (ixdy) [inline pkg/util/mount/mount_unsupported.go]: not knowing the context this is used is, is `true` a reasonable default?

Comment (rootfs) [inline pkg/util/mount/mount_unsupported.go]: true is not a bad default (vs. false), it prevents others using the mount point.

**Comment by saad-ali** [comment]
LGTM
Thanks for fixing @ixdy.

**Comment by ixdy** [comment]
I didn't fix, I just pointed out the breakage. :)

**Comment by saad-ali** [comment]
Ah yes, I meant thanks @rootfs

**Pull request merged**